ui/src/pages/insightDetail/components/eventAggregator/styles.module.less

```diff
@@ -1,10 +1,7 @@
 .events_container {
   display: flex;
   flex-direction: column;
```

Comment:
Removing the background color, border-radius, and box-shadow may affect the visual design. Consider if these properties are still needed for consistency.